packages/libs/src/sdk/api/albums/AlbumsApi.ts

```diff
@@ -358,7 +360,8 @@ export class AlbumsApi {
           locationMemoInstruction
         ],
         priorityFee: {
-          microLamports: 100000
+          priority: 'VERY_HIGH',
```

Comment:
Just thinking about SDK usage outside of client: This feels like something that should be configurable by the end user if possible. I'm fine with it defaulting to very high, but if I were building something programmatic using the SDK, I would want the ability to set this low (i.e. maybe I write a scheduler that attempts the purchase later when fees are low).

Reply:
while this is spiritually the right call - i think it's bike shed. the fees are fairly negligible.
150,000 microlamports * ~100,000 CU's per tx = 15k lamports
15k lamports = 15,000 / 1,000,000,000 * $160 = $0.0024

i think we should be very careful to introduce complexity to this code path because keeping fees on is very beneficial for us.

Reply:
I think I've found a decent middle ground - going to have the `BaseSolanaProgramClient` default to `'VERY_HIGH'` w/ 150k microLamports per CU. If a 3rd party wants to configure that differently, they can override the client and change the default there, which they're already probably doing similar if they're moving away from using our relay. But the API callers don't need to specify if they're all going to choose the same priority, and we can prevent shooting ourselves in the foot later by not including it on accident.